src/app.ts

```diff
@@ -13,14 +13,38 @@ interface TemperatureSummary {
   average: number
 }
 
+const dataByCity: { [key: string]: TemperatureReading[] } = {}
```

Comment:
Thank you for the observation, I found working with the global object a little bit easier for this challenge, but it's true that this can cause unexpected behavior in real world scenarios.